Settings.cs

```diff
@@ -431,6 +431,69 @@ public static Settings Instance
         [SettingProperty("Probability to deliver twins", -1.0f, 1.0f, "Native value is 0.03. Determines the chance of giving birth to twins.")]
         [SettingPropertyGroup("Pregnancy Tweaks/Twins Probability Tweak")]
         public float TwinsProbability { get; set; } = 0.03f;
+        [XmlElement]
+        [SettingProperty("Enable Character Fertility Probability Tweak", "Allows for adjusting for the probability to get pregnant, this will apply to everyone. By enabling this, we can use Min/Max Pregnancy Age")]
+        [SettingPropertyGroup("Pregnancy Tweaks/Fertility Probability Tweak", true)]
+        public bool CharacterFertilityProbabilityTweakEnabled { get; set; } = false;
+        [XmlElement]
+        [SettingProperty("Character Fertility Probability", 0f, 1.0f, "Native value is 0.95. Determines the chance of getting pregnant")]
+        [SettingPropertyGroup("Pregnancy Tweaks/Fertility Probability Tweak")]
+        public float CharacterFertilityProbability { get; set; } = 0.95f;
+        [XmlElement]
+        [SettingProperty("Enable Daily Chance Pregnancy Tweak", "Enabling this will completely override the daily pregnancy check. All settings below will be applied!")]
+        [SettingPropertyGroup("Pregnancy Tweaks/Daily Chance Pregnancy Tweak", true)]
+        public bool DailyChancePregnancyTweakEnabled { get; set; } = false;
+        [XmlElement]
+        [SettingProperty("Enable Player Character Fertility", "Is the player character (you) fertile? Native: true")]
+        [SettingPropertyGroup("Pregnancy Tweaks/Daily Chance Pregnancy Tweak")]

```

Comment:
Cleanup the subgroup names, all of them are Daily Chance Pregnancy Tweak.

Reply:
Ignore this, I see where you going with this now :)

Reply:
all of them are part of the dail chance pregnancy tweak because I override the method calculating it.
I am not adjusting the calculation but I adding a few conditions before it gets calculated